homeassistant/components/system_bridge/sensor.py

```diff
@@ -0,0 +1,344 @@
+        )
+
+    @property
+    def device_state_attributes(self) -> dict[str, Any]:
+        """Return the state attributes of the entity."""
+        bridge: Bridge = self.coordinator.data
+        return {
+            ATTR_AVAILABLE: bridge.filesystem.fsSize[self._key]["available"],
+            ATTR_FILESYSTEM: bridge.filesystem.fsSize[self._key]["fs"],
+            ATTR_MOUNT: bridge.filesystem.fsSize[self._key]["mount"],
+            ATTR_SIZE: bridge.filesystem.fsSize[self._key]["size"],
+            ATTR_TYPE: bridge.filesystem.fsSize[self._key]["type"],
+            ATTR_USED: bridge.filesystem.fsSize[self._key]["used"],
+        }
+
+
+class BridgeKernelSensor(BridgeSensor):
+    """Defines a kernel sensor."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, bridge: Bridge):
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            bridge,
+            "kernel",
+            "Kernel",
+            "mdi:devices",
+            None,
+            None,
+            True,
+        )
+
+    @property
+    def state(self) -> str:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return bridge.os.kernel
+
+
+class BridgeOsSensor(BridgeSensor):
+    """Defines an OS sensor."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, bridge: Bridge):
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            bridge,
+            "os",
+            "Operating System",
+            "mdi:devices",
+            None,
+            None,
+            True,
+        )
+
+    @property
+    def state(self) -> str:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return f"{bridge.os.distro} {bridge.os.release}"
+
+
+class BridgeProcessesLoadSensor(BridgeSensor):
+    """Defines a Processes Load sensor."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, bridge: Bridge):
+        """Initialize System Bridge sensor."""
+        super().__init__(
+            coordinator,
+            bridge,
+            "processes_load",
+            "Load",
+            "mdi:percent",
+            None,
+            PERCENTAGE,
+            True,
+        )
+
+    @property
+    def state(self) -> float:
+        """Return the state of the sensor."""
+        bridge: Bridge = self.coordinator.data
+        return (
+            round(bridge.processes.load.currentLoad, 2)
+            if bridge.processes.load.currentLoad is not None
+            else None
+        )
+
+    @property
+    def device_state_attributes(self) -> dict[str, Any]:
+        """Return the state attributes of the entity."""
+        bridge: Bridge = self.coordinator.data
+        return {
+            ATTR_LOAD_AVERAGE: round(bridge.processes.load.avgLoad, 2)
+            if bridge.processes.load.avgLoad is not None
+            else None,
```

Comment:
Our standard is to not include the item at all in the state attributes if the value is `None`.